circe/src/main/scala/org/http4s/circe/CirceInstances.scala

```diff
@@ -115,6 +115,11 @@ trait CirceInstances extends JawnInstances {
       Uri.fromString(str).leftMap(_ => "Uri")
     }
 
+  implicit class JsonDecoderSyntax[F[_]](private val req: Message[F])(implicit F: JsonDecoder[F]) {
```

Comment:
Is it worthwhile to add `extends AnyVal` to, as I understand, minimize allocations?

Reply:
Nevermind: https://github.com/http4s/http4s/pull/2907#discussion_r333745452.